Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml

```diff
@@ -1,69 +1,80 @@
 ﻿- type: entity
   abstract: true
-  parent: Clothing
+  parent: ClothingHeadset
   id: ClothingHeadsetAlt
   name: headset
   description: An updated, modular intercom that fits over the head. Takes encryption keys.
   components:
-  - type: Headset
   - type: Sprite
     state: icon_alt
   - type: Clothing
     equippedPrefix: alt
-    slots:
-    - ears
+
+- type: entity
+  parent: ClothingHeadsetAlt
+  id: ClothingHeadsetAltCargo
+  name: qm over-ear headset
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyQM
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/cargo.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/cargo.rsi
```

Comment:
The sprite for this isn't going to be an over-ear headset.

Reply:
Look in parent
```yml
- type: Sprite
    state: icon_alt
```